internal/utils_test.go

```diff
@@ -253,6 +253,156 @@ data:
 	}
 }
 
+func TestGetPolicyTemplateKustomize(t *testing.T) {
+	t.Parallel()
+	kustomizeDir := t.TempDir()
+	configStrings := []string{"tomato", "potato"}
+
+	manifestsDir := path.Join(kustomizeDir, "manifests")
```

Comment:
What is `manifestsDir` used for in this test?

Reply:
It's the directory called in the Kustomize file.